shell.c

```diff
@@ -69,6 +76,13 @@ static int subcommand_usage(const char* name) {
         "  --grammar-str/-g FILE\tpeg grammar string\n"
         "  --grammar-file/-G FILE\tpath to peg grammar file\n"
         "  --grammar-entry/-e NAME\tentry rule name in peg grammar\n"
+#ifndef WITHOUT_DEBUG_INFO
+        "  --debug/-d\t\toutput debug info\n"
+#endif
+        "  --json/-j\t\tjson ast output\n"
+        "  --json2/-J\t\tjson ast output with only arrays\n"
```

Comment:
`--json2` is a little bit confusing and does not clearly specify the purpose of the option.

How about `--format=jsonarray`?